app/src/main/java/com/quran/labs/androidquran/ui/translation/TranslationAdapter.kt

```diff
@@ -432,7 +432,7 @@ internal class TranslationAdapter(private val context: Context,
 
   companion object {
     private val USE_UTHMANI_SPAN = Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP
-    private const val ARABIC_MULTIPLIER = 1.4f
```

Comment:
making this public so SeekBarTextSizePreference can show an accurate preview of arabic text size